stacks/spica/src/function/examples/examples.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Can we use a typescript file instead of json and format these examples beforehand?